linkerd/announcer/serversets/src/main/scala/io/l5d/announcer/Serversets.scala

```diff
@@ -0,0 +1,13 @@
+package io.l5d.announcer
+
+import com.twitter.finagle.Announcer
+import com.twitter.finagle.zookeeper.buoyant.ZkAnnouncer
+import io.buoyant.linkerd.{AnnouncerInitializer, AnnouncerConfig}
+
+class ServersetsAnnouncerInitializer extends AnnouncerInitializer {
+  override def configClass = classOf[Serversets]
+}
+
+case class Serversets(hosts: Seq[String], pathPrefix: Option[String]) extends AnnouncerConfig {
+  override def mk(): Announcer = new ZkAnnouncer(hosts.mkString(","), pathPrefix.getOrElse("/discovery"))
```

Comment:
tioli: I like defining defaults as constants in a companion class vs. having them hard-coded in the code, but obviously it's not a huge deal in this case where it's only used once.